dbt_subprojects/daily_spellbook/models/_metrics/transfers/metrics_net_solana_transfers.sql

```diff
@@ -0,0 +1,113 @@
+{{ config(
+        schema = 'metrics'
+        , alias = 'net_solana_transfers'
+        , materialized = 'incremental'
+        , file_format = 'delta'
+        , incremental_strategy = 'merge'
+        , unique_key = ['blockchain', 'block_date', 'tx_hash']
+        , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+        )
+}}
+
+with raw_transfers as (
+    /*
+    - get the transfer amount sent per address in a transaction hash, set the amount sent as a negative value
+    - union the transfer amount received per address in a transaction hash, keep the amount received as a positve value
+    */
+    select
+        blockchain
+        , block_date
+        , tx_id as tx_hash
+        , from_base58(from_owner) as address
```

Comment:
is `from_base58()` needed, since solana is separate model here?

Reply:
nope, would leave it as base58 instead